### PR TITLE
Adding submit review and view functionality #14

### LINKED ISSUE #14: As a user, I want to filter different services (by rating or distance) to choose the most suitable option

### LINKED ISSUE #52: As a service seeker, I would like to leave reviews about a service, so that my feedback can heard by providers and give guidance to other users on the platform.

### LINKED ISSUE #53: As a service seeker, I would like to see reviews about a service, in order to make an informed decision on whether I would use that service.

### PR DESCRIPTION
**Pull request by shub-garg**

### CODE REVIEW

**Comment by deepjyotk** [comment]
please resolve the conflicts

File: db-prep/db-reviews.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        table.meta.client.get_waiter('table_exists').wait(TableName=table_name)
+        print(f"Table '{table_name}' has been created successfully.")
+        return table
+
+    except ClientError as e:
+        print(f"Failed to create table: {e.response['Error']['Message']}")
+        sys.exit(1)
+
+def main():
+    dynamodb = boto3.resource('dynamodb', region_name='us-east-1')  # Replace with your region
```
Comment (KartikeySharma) [inline db-prep/db-reviews.py]: nit: let's remove these comments...not a blocker

**Review by KartikeySharma — APPROVED** [review]

**Comment by KartikeySharma** [comment]
image
 Functionality works!

**Comment by KartikeySharma** [comment]
pls resolve the conflicts, LGTM!

**Review by omerfarukbasar — APPROVED** [review]
Tested yesterday locally. Review viewing and submission features worked.

**Pull request merged**